sdk/identity/identity/src/msal/flows.ts

```diff
@@ -19,6 +19,7 @@ export interface MsalFlowOptions {
   disableAutomaticAuthentication?: boolean;
   disableInstanceDiscovery?: boolean;
   getAssertion?: () => Promise<string>;
+  enableMsaPassthrough?: boolean;
```

Comment:
Note the difference in casing used here. This is the casing I'd expect to see for the property.